universal-application-tool-0.0.1/app/repository/AmazonS3Client.java

```diff
@@ -92,6 +98,20 @@ public void putObject(String key, byte[] data) {
     }
   }
 
+  public URL getPresignedUrl(String key) {
+    GetObjectRequest getObjectRequest = GetObjectRequest.builder().key(key).bucket(bucket).build();
+
+    GetObjectPresignRequest getObjectPresignRequest =
+        GetObjectPresignRequest.builder()
+            .signatureDuration(Duration.ofMinutes(10))
```

Comment:
suggestion (blocking): extract `Duration.ofMinutes(10)` to a constant

Reply:
Done.